### DISCUSSION TITLE
Is there a way in WalletConnect to let other services know the URL of our own relay server?

### DISCUSSION THREAD

**Discussion by jaripy** [WalletConnect/walletconnect-monorepo#1729] [Developer Support]
We are planning to create our own wallet and consider to build our own relay server.
If we create our own relay server, is there a way to register the relay server URL in WalleConnect so that it is visible from other DApps?
Wo would like to know if there is any other way other than telling the URL directly to the DApps or the wallet administrators.

Thank you for your cooperation.

**Comment by chadyj** [comment]
We don't yet support self-hosting your own v2 relay.
This feature is coming in 2023.

**Reply by jaripy** [reply]
Thank you.

If you don’t mind, may I ask when you plan to support it in 2023?
